src/shared/utils/mixpanel/mixpanel.utils.ts

```diff
@@ -27,40 +27,98 @@ export const trackAppletSave = ({
 }) => {
   if (!applet) return;
 
-  const itemTypes: ItemResponseType[] = [];
+  let itemCount = 0;
+  let phraseBuilderItemCount = 0;
+  let itemsIncludedInPhraseBuilders = 0;
+  const phraseBuilderFieldCounts = {
+    itemResponse: 0,
+    sentence: 0,
+    lineBreak: 0,
+  };
   let autoAssignedActivityCount = 0;
   let manualAssignedActivityCount = 0;
-  for (const activity of applet.activities ?? []) {
-    itemTypes.push(...(activity.items ?? []).map((item) => item.responseType));
+  const itemTypes: ItemResponseType[] = [];
```

Comment:
This array seems to only be used as the argument to a Set constructor. Maybe we could just make this a Set